### ISSUE TITLE
[Mobile Accessibility at W3C] fix missing indication of "in english" for some links

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-translations#174]
e.g., see some there and some missing under https://www.w3.org/WAI/standards-guidelines/mobile/es#covered

That entails fixing brackets around links:
* [text that is linked](/path/filename/)
* [[Title of WAI Page]](/path/filename/)

([[Title of WAI Page]] -- when it is the exact title, then it pulls the translated title if it's available)

will need to correct that in the English, then all translations. :-/

**Comment by remibetin** [comment]
@shawna-slh I do not see anything out of the ordinary in the code or rendering of the resource.

The only links without "In English" are:
- Links pointing outside WAI Website
- Links to Working Groups pages (not supported globally, not specific to this resource and nothing we can do short-term)

And https://github.com/w3c/wai-mobile/commit/e366bc9ac76dc509c22ca1f994ddac6dc077c723 has probably fixed the only problematic link.

I suggest to close this issue.

**Comment by remibetin** [comment]
Can't reproduce
